tst/EnergyPlus/unit/CommandLineInterface.unit.cc

```diff
@@ -385,8 +385,17 @@ TEST_F(CommandLineInterfaceFixture, runReadVars)
     }
 }
 
-TEST_F(CommandLineInterfaceFixture, DISABLED_numThread)
+TEST_F(CommandLineInterfaceFixture, numThread)
 {
+
+    // This test fails on GitHub Actions CI and I am not sure why.
+    // I still want it exercised regularly, so I am going to let CI
+    // just wrap it up early and pass.  This test has spent most of its
+    // life DISABLE_d anyway, so this is a net improvement.
+    if (std::getenv("CI")) {
+        return;
+    }
+
```

Comment:
Yeah, this test passes happily on all my machines.  Something about CI is different.  Maybe .... hmm I wonder if it's because the CI image doesn't have enough threads?  Hmm... @jmarrec what do you think?